### DISCUSSION TITLE
Is it possible to use framer-motion v6+ in ES5 browsers?

### DISCUSSION THREAD

**Discussion by Noitidart** [motiondivision/motion#1953] [Q&A]
Framer-motion v6+ seems to ship non-es5 compatible code. Is it possible to transpile this down so there are no `let`s or `...`s? I'm using Next.js 13.1 and used the `transpilePackages` feature seen here - https://nextjs.org/docs/advanced-features/compiler#module-transpilation

While I have `browserslist` for `">0.3%, defaults, supports es5"` however I get an error:

```
Failed to compile.

./node_modules/framer-motion/dist/es/render/dom/value-types/type-int.mjs
Module parse failed: Export 'int' is not defined (7:9)
File was processed with these loaders:
 * ./node_modules/next/dist/build/webpack/loaders/next-swc-loader.js
You may need an additional loader to handle the result of these loaders.
|     transform: Math.round
| });
> export { int };
| 
```

I also tried using non-Next.js to transpile it, and got same error. Is it possible to transpile this down?

**Comment by farzadabroudi** [comment]
I have this problem too
**Help**

**Reply by Noitidart** [reply]
I was using next.js which and had framer-motion in trasnpilePackages in next.config.js - but next.js had a bug in the SWC until next.js 14.0.4 - once upgrading to next.js 14.0.4 and transpilePackages works right for framer-motion.
